features/lege-waarden.feature

```diff
@@ -173,15 +252,15 @@ Functionaliteit: Leveren van lege waarden
       Dan heeft de response een persoon met een 'kind' met een leeg 'geboorte' object
       # "kinderen": [{ "geboorte": {} }]
 
-    Scenario: Kind met geboorteplaats alleen omschrijving en vragen om code geeft geboorteplaats als leeg object
+    Scenario: Kind heeft wel geboorteplaats omschrijving en geen geboorteplaats code
       Gegeven het systeem heeft een persoon met de volgende gegevens
       | naam                | waarde    |
       | burgerservicenummer | 555550003 |
       En de persoon heeft een 'kind' met alleen de volgende 'geboorte' gegevens
-      | naam   | waarde   |
-      | datum  | 20070421 |
-      | plaats | Berlijn  |
-      | land   | 0000     |
+      | naam                  | waarde   |
+      | geboortedatum (03.10) | 20070421 |
+      | plaats.omschrijving   | Berlijn  |
```

Comment:
Ik begrijp niet goed hoe een geboorte geen geboorteplaats code zou kunnen hebben en wel een omschrijving. Volgens het LO GBA bevat geboorte de volgende velden:

03.10 Geboortedatum
03.20 Geboorteplaats
03.30 Geboorteland

Als je dan gaat kijken wat er in Geboorteplaats zit dan staat daar:

Een code, opgenomen in Tabel 33, Gemeententabel of een buitenlandse plaats of een plaatsbepaling, die aangeeft waar de persoon is geboren.

Ik zie dus geen mogelijkheid om in de BRP alleen een omschrijving vast te leggen

Ik begrijp ook niet waarom je in de regels 261 en 263 verwijst naar een LO GBA nummer (03.10 en 03.30) en dat in regel 262 niet doet.